src/_posts/hands-on-i.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Hands on! Building interactive training with A-Frame
+author: twitter|salvadelapuente|Salvador de la Puente
+date: 2017-07-26 12:00:00
+layout: blog
+image:
+  src: hands-on-i.png
+---
+
+In an industry where manipulation of complex mechanisms involves dozens of tools, expensive equipment and even safety risks, virtual reality helps to reduce material costs and increase operator safety. In addition, WebVR puts everything just one link away.
```

Comment:
`s/In addition, WebVR puts everything just one link away/With WebVR, everything is one link away`